client/src/components/home/carousel.tsx

```diff
@@ -1,93 +1,69 @@
-  //   name: 'Julien Demenois, France',
-  //   role: 'Policymaker',
-  // },
-  // {
-  //   id: 3,
-  //   Icon: Landmark,
-  //   content:
-  //     '"Thanks to Impact4Soil I will be able to quickly identify relevant scientific publications on soil carbon, and even better, I will have a quantified assessment of the impact of practices on soil carbon. This last point is very important for me, as part of a research organisation, because we are often asked to provide this information by decision-makers and journalists. Knowing that there is a reliable source of scientific information is therefore extremely valuable"',
-  //   name: 'Julien Demenois, France',
-  //   role: 'Funding agency',
-  // },
-  // {
-  //   id: 4,
-  //   Icon: Factory,
-  //   content:
-  //     '"Thanks to Impact4Soil I will be able to quickly identify relevant scientific publications on soil carbon, and even better, I will have a quantified assessment of the impact of practices on soil carbon. This last point is very important for me, as part of a research organisation, because we are often asked to provide this information by decision-makers and journalists. Knowing that there is a reliable source of scientific information is therefore extremely valuable"',
-  //   name: 'Julien Demenois, France',
-  //   role: 'Company',
-  // },
-  // {
-  //   id: 5,
-  //   Icon: HeartHandshake,
-  //   content:
-  //     '"Thanks to Impact4Soil I will be able to quickly identify relevant scientific publications on soil carbon, and even better, I will have a quantified assessment of the impact of practices on soil carbon. This last point is very important for me, as part of a research organisation, because we are often asked to provide this information by decision-makers and journalists. Knowing that there is a reliable source of scientific information is therefore extremely valuable"',
-  //   name: 'Julien Demenois, France',
-  //   role: 'NGO',
-  // },
-];
+const iconMap: Record<string, LucideIcon> = {
+  microscope: Microscope,
+  scale: Scale,
+  landmark: Landmark,
+  factory: Factory,
+  'heart handshake': HeartHandshake,
+};
 
 const cardVariants = {
   hidden: { opacity: 0, x: '-100%' },
   visible: { opacity: 1, x: 0 },
 };
 
 type CardProps = {
-  card: (typeof cards)[0];
+  card?: Testimony;
 };
 
 const Card = ({ card }: CardProps) => {
-  const { Icon, content, name, role } = card;
+  if (!card) return null;
+  const { icon, content, name, role } = card;
+  const Icon = iconMap[icon];
+
   return (
     <motion.div
-      key={card.id}
       className="flex h-[444px] min-w-[373px] max-w-[373px] flex-col justify-between gap-6 border-b-8 border-yellow-500 bg-white p-10 shadow"
       variants={cardVariants}
     >
       <div className="flex h-12 w-12 items-center justify-center bg-gray-700 p-2">
         <Icon className="h-12 w-12 text-white" />
       </div>
-      <p className="text-sm leading-6 text-gray-700">{content}</p>
+      <p
```

Comment:
This element must be a `div` otherwise we'll insert `p`s inside of `p`s which is invalid HTML.